runtime/lua/vim/health/health.lua

```diff
@@ -397,6 +397,23 @@ local function check_external_tools()
   end
 end
 
+local function check_net()
+  health.start('External Tools (vim.net)')
+
+  if vim.fn.executable('curl') == 1 then
+    local curl = vim.fn.exepath('curl')
+    local cmd = { 'curl', ' --version' }
+    local result = vim.system(cmd, { text = true }):wait()
+
+    health.ok(('%s\n(%s)'):format(vim.trim(result.stdout), curl))
```

Comment:
For curl, we _really_ need the full output (including supported protocols and features!), as well as the location (in case people have multiple curl versions installed); see https://github.com/nvim-treesitter/nvim-treesitter/blob/36aee737214f5577d8b5c7715067f2498db97a7b/lua/nvim-treesitter/health.lua